interfaces/matlab_experimental/Reactor/ReactorNet.m

```diff
@@ -54,46 +54,57 @@
     methods
         %% ReactorNet Class Constructor
 
-        function r = ReactorNet(reactors)
+        function n = ReactorNet(r)
             % Create a :mat:class:`ReactorNet` object.
 
             ctIsLoaded;
 
-            if nargin ~= 1
-                error('Wrong number of arguments to ReactorNet constructor.');
+            if nargin == 0
+                r = {};
             end
 
-            if isa(reactors, 'Reactor')
+            if isa(r, 'Reactor')
                 % Allow simpler syntax for creating a network with one
                 % reactor.
-                reactors = {reactor};
+                r = {r};
```

Comment:
I believe that single-letter variable names should be avoided except for simple cases (iterators).